### ISSUE TITLE
New Religions

### ISSUE THREAD

**Issue by matthewh86** [matthewh86/NoxModTheOldGods#10]
**Bold** religions are religion families. Each religion is still unique, but religion families allow Royal Marriage between members.

**American**
- [x] Reformed Aztec
  - [ ] _Aztec Pagan_
- [x] Reformed Inca
  - [ ] _Inca Pagan_
- [ ] Reformed Cherokee
  - [ ] _Cherokee Pagan_

**Dharmic**
Hinduism is in this family
- [x] Jainism
  - [x] _Jamalism_

**Druidic**
- [ ] Druidism
  - [ ] _Wicca_

**Hellenic**
- [ ] Hellenism
  - [ ] _Gaiaism_

**Nordic/Norse**
- [x] Germanic Norse
  - [x] _Wotansrabe_
- [x] Scandinavian Norse
  - [x] _Odinshrafn_
- [x] Suomenusko
  - [x] _Jumism_

**Slavic**
- [x] Romuva
  - [x] _Old Romuva_
- [x] Slavic
  - [x] _Old Slavic_
- [x] Tengri
  - [x] _Old Tengri_

**Zoroastrian**
- [x] Zoroastrianism
  - [ ] _Yazidi_
- [x] Zunism
  - [ ] _Messalian_

**Comment by BillNyeTheRussianSpy** [comment]
Baltic should be Romuva

**Comment by matthewh86** [comment]
Yup, will change. Couldn't remember the name!

**Comment by BillNyeTheRussianSpy** [comment]
Heresies:
Key:
Religion:(notes)
Heresy(notes)

Zoroastrian:(should buff this religion)
Manichaean(should be very unique because they borrowed ideas of many religions)
Mazdaki(look up this religion)

Orthodoxy:(should have a religious head that can excommunicate)
Bogomilist(shouldn't have a patriarchy system and can have female religious rulers)
Monothelite(should have a religious head that can excommunicate and have patriarchy system)
Iconoclast(should have a religious head that can excommunicate and have patriarchy system)
Paulician(should have a religious head that can excommunicate and have patriarchy system)

Catholicism:(no change needed)
Lollard(look up this religion)
Cathar(there can be female religious leaders, and queens can become generals)
Fraticelli(should have a pope once established)
Waldensian(look up this religion)

Miaphysite:(Called Coptic in Eu4, we should change that around, and should have the patriarchy system)
Monophysite(should have the patriarchy system and look up this religion)

Nestorian:(should have the patriarcy system, but messalian shouldn't and messalians should have female religious rulers)
Messalian (Nestorian Heresy influenced by Zoroastrians so they have Divine Blood in Ck2 too.)

Judaism:(no change needed)
Samaritan (look up this religion)
Karaite(look up this religion)

Sunni:(no change needed)
Zikri(look up this religion)
Yazidi(should have excommunication and look up this religion)

Ibadi:(no change needed)
Kharijite (look up this religion)

Shia:(no change needed)
Druze(look up this religion)
Hurufi(look up this religion)

We should borrow most of the religious mechanics and bonuses from ck2,and the piety and patriarchy systems should be re-buffed.http://www.ckiiwiki.com/Religion Religions that should have female religious rulers also have male ones too :P When I mean by look it up ckeck the ck2 wiki and look up the religion on the internet and give the sources of the new info so I can check its validity.

**Comment by BillNyeTheRussianSpy** [comment]
All religions that had religious heads in real should exist in the mod in a certain form like the Catholic religious head. And in general add more religious flavor to all religions. and add more of them.
And you for heresies you should be able to convert anyway ex. Orthodoxy->Iconoclast Iconoclast->Orthodoxy Iconoclast->Monothelite 
And each religious group should have a unite (religion group), discuss later)
And all reglions that have religious heads can declare Crusader,Jihads, and Great holy wars on nations like how the cathlocs do it, except now all nations of that religion get the holy war cb and when that is used reigionn of the defender and attacker can join respectably and during these nations are great huge bonuses so these wars are encouraged.

**Comment by BillNyeTheRussianSpy** [comment]
Pagans:
All pagans including Zunists, Hellenic and Generic Pagan(http://www.ckiiwiki.com/Religion#List_of_religions) should be able to reform. when this happens a schism will happen that you can mend a while LATER. When this happens well lets take an example.
When Zunists reform they split into Reformed Zunist and Old Zunist like ck2. and Hindus, Buddhists, and Zoroastrians and maybe there hersies can convert to Zunist(or refomed and old after reformation), all reforomed pagan should have a religious head. All Pagans can convert to Generic Pagan and the reformed religious heads of Hellenics and Generic Pagans will be called High Priests.
Whenever the Reformed pagan religion will have reformed in front of it is still to be debated.
So to sum up, religion will be huge.

**Comment by BillNyeTheRussianSpy** [comment]
*BUMP FOR NEW RELIGIOUS THINGS*

**Comment by matthewh86** [comment]
Most of what you have said will either be too much work:
- 22 extra religions
- reforming religions

Or as far as I know is impossible:
- Papal mechanics for non-Catholics
- Excommunication for non-Catholics
- Religious heads
- Religious heads as state entities
- Defenders of the Faith to join an aggression war
- Female generals

Or already exists in the game in another form:
- Holy War CB already exists in the Religious Ideas group

Some of what you said can be done however:
- Patriarch authority for Coptic/Miaphysite
- Defenders of the Faith for other religions

**Comment by matthewh86** [comment]
Making a new Papacy seems possible, but a lot of work. It might get done later.
